### PR TITLE
Start analysis http codes

### PR DESCRIPTION
**Pull request by tisnik**
We need to test the authentication module as well, so we can accept HTTP 401 and check it later.

### CODE REVIEW

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -109,9 +110,9 @@ def start_analysis_for_component(context, ecosystem, component, version):
     if response.status_code == 200:
         raise Exception('Bad state: the analysis for component has been '
                         'finished already')
-    elif response.status_code != 404:
-        raise Exception('Improper response: expected HTTP status code 404, '
-                        'received {c}'.format(c=status_code))
+    elif response.status_code != 401 and response.status_code != 404:
```
Comment (jpopelka) [inline integration-tests/features/steps/common.py]: or
`elif response.status_code not in {401, 404}:`

Comment (tisnik) [inline integration-tests/features/steps/common.py]: Thans @jpopelka . I suppose you meant (401, 404), not {401, 404}, right?

**Comment by jpopelka** [comment]
I really meant `{401, 404}`, i.e. a set literal, but in this case it's ok to use `(401, 404)`, i.e. a tuple literal.

```python
In [1]: type({401, 404})
Out[1]: set

In [2]: type((401, 404))
Out[2]: tuple
```

**Pull request merged**